frontend/src/components/SearchByArticlesList.js

```diff
@@ -26,10 +27,12 @@ const SearchByArticlesList = () => {
         <div key={index} className="card">
           <div className="card__body">
             <div className="article-item">
-              <img alt="" className="avatar--md" src={article.thumbnail} />
-              <div>
-                <strong>{article.title}</strong>
-              </div>
+              <Link to={`/article/${article.id}`}>
+                <img alt="" className="avatar--md" src={article.thumbnail} />
```

Comment:
Is it worth adding an alt text here to be more accessible?
```suggestion
                <img alt="article thumbnail" className="avatar--md" src={article.thumbnail} />
```